packages/webapp/src/components/Animals/RemoveAnimalsModal/index.tsx

```diff
@@ -70,39 +72,58 @@ export default function RemoveAnimalsModal(props: RemoveAnimalsModalProps) {
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
 
+  // Reset form when a new set of animals/batches is selected
+  useEffect(() => {
+    if (props.isOpen) {
+      reset({ [DATE]: getLocalDateInYYYYDDMM() });
+    }
+  }, [props.isOpen]);
```

Comment:
If we want to reset the form to default every time the modal's open, I think this can be achieved by passing `shouldUnregister: true` to `useForm`

Reply:
Oh nice, thank you! I didn't know that reset to default value!